src/lib/slider/slider.scss

```diff
@@ -214,7 +214,7 @@ $mat-slider-focus-ring-size: 30px !default;
 
 
 // Active slider.
-.cdk-focused {
+.mat-slider.cdk-focused:not(.mat-slider-disabled) {
```

Comment:
this adds a lot off specificity, are you sure it doesn't break any of the other states?

Reply:
I didn't notice anything from poking around the demo. I can reduce the specificity slightly by changing the selector to `.cdk-focused:not(.mat-slider-disabled)`.

Reply:
I think its good to leave the `.mat-slider` so it doesn't wind up triggering when inside of another focused element, just curious if you had checked for other cases because I know I've had CSS regressions in the past with slider when I try to do these kind of changes

Reply:
One place I could see it potentially breaking is the mouse, keyboard etc. focused styles a bit below. I can scope those to `.mat-slider:not(.mat-slider-disabled)` as well.

Reply:
Reworked as discussed @mmalerba.